pkg/sync/v7/data/DaemonSet.apps/openshift-azure-logging/omsagent.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    azure.openshift.io/sync-pod-optionally-apply: '*** GENERATED ***'
+    azure.openshift.io/sync-pod-wait-for-readiness: "false"
+  name: omsagent
+  namespace: openshift-azure-logging
+spec:
+  selector:
+    matchLabels:
+      app: omsagent
+  template:
+    metadata:
+      labels:
+        agentVersion: 1.10.0.1
+        app: omsagent
+        dockerProviderVersion: 6.0.0-0
+    spec:
+      containers:
+      - env:
+        - name: WSID
+          value: '*** GENERATED ***'
+        - name: KEY
+          value: '*** GENERATED ***'
+        - name: AKS_RESOURCE_ID
+          value: '*** GENERATED ***'
+        - name: AKS_REGION
+          value: '*** GENERATED ***'
+        - name: CONTROLLER_TYPE
+          value: DaemonSet
+        - name: NODE_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.hostIP
+        image: '*** GENERATED ***'
+        livenessProbe:
+          exec:
+            command:
+            - /bin/bash
+            - -c
+            - /opt/livenessprobe.sh
+          initialDelaySeconds: 60
+          periodSeconds: 60
+        name: omsagent
+        ports:
+        - containerPort: 25225
+        - containerPort: 25224
+          protocol: UDP
+        resources:
+          limits:
+            cpu: 150m
+            memory: 600Mi
+          requests:
+            cpu: 75m
+            memory: 225Mi
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /hostfs
+          name: host-root
+          readOnly: true
+        - mountPath: /var/run/host
+          name: docker-sock
+        - mountPath: /var/log
+          name: host-log
+        - mountPath: /var/lib/docker/containers
+          name: containerlog-path
+        - mountPath: /etc/kubernetes/host
+          name: azure-json-path
+        - mountPath: /etc/config
+          name: omsagent-config
+        - mountPath: /etc/config/settings
+          name: settings-vol-config
+          readOnly: true
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+        node-role.kubernetes.io/compute: "true"
```

Comment:
only run on the compute node until we have figured out how to do audit logs (let's do that as a later work item).